rasa/core/policies/ted_policy.py

```diff
@@ -500,6 +500,10 @@ def load(cls, path: Union[Text, Path]) -> "TEDPolicy":
         )
         model.build_for_predict(predict_data_example)
 
+        meta["should_finetune"] = kwargs.get("should_finetune", False)
+        if "epoch_override" in kwargs:
+            meta[EPOCHS] = kwargs["epoch_override"]
+
```

Comment:
This needs a larger change which i'll address in my PR as part of making core policies finetunable. It's okay to keep as it is here.